code/modules/projectiles/guns/special/hand_of_midas.dm

```diff
@@ -0,0 +1,130 @@
+// Hand of Midas
+
+/obj/item/gun/magic/midas_hand
+	name = "The Hand of Midas"
+	desc = "An ancient Egyptian matchlock pistol imbued with the powers of the Greek King Midas. Don't question the cultural or religious implications of this."
```

Comment:
This is even more ironic if you take in consideration Midas was actually Phrygian (from Anatolia/Turkey).